api/sql/schema/750-rule.decision.lookup.sql

```diff
@@ -75,7 +75,7 @@ BEGIN
     FROM
         [integration].[vAccount]
     WHERE
-        accountNumber = @sourceAccount AND
+        (accountNumber = @sourceAccount OR @sourceAccount IS NULL) AND
```

Comment:
Allow calling with account owner only, as there may be no accounts at all and rule still can be used

Reply:
then may be this should be added:
 (@sourceAccountOwnerId IS NOT NULL OR @sourceAccount IS NOT NULL)
otherwise it can be called bit both parameters = NULL